src/components/LandUseVocabulary.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import _ from 'lodash'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import Tooltip from 'react-bootstrap/Tooltip'
+import Button from 'react-bootstrap/Button'
+
+import terms from '../assets/keywords'
+
+import './LandUseVocabulary.css';
+
+class LandUseVocabulary extends Component {
+  
+  state = {
+    usesString: this.props.string,
+  };
+
+  render() {
+    const { usesString } = this.state;
+    // splitting the uses
+    let uses = _.split(usesString, ',')
+    //return <div dangerouslySetInnerHTML={this.createMarkup(this.state.html)} />;
+    return uses.map(use=>this.renderUse(use))
+    return 'gal'
+  }
+  renderUse(use) {
+    let useTerm = this.finduse(use);
+
+    if(useTerm && useTerm.description){
```

Comment:
It's better not to nest the code if we don't have to.
```suggestion
    if(!useTerm || !useTerm.description){
    return null
    }
```

Reply:
done